<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>IGNQt</string>
  </property>
  <widget class="QWidget" name="wgt_central">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="wgt_stacked">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pnl_empty">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QClickDropLabel" name="lbl_open_image">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Click inside to open the Filebrowser&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;or drop an Image anywhere&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="dropRejectText" stdset="0">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align='center'&gt;&lt;span style='font-size:14pt;'&gt;Please drop a &lt;strong&gt;single&lt;/strong&gt; PNG or JPEG&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pnl_view">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QImageLabel" name="lbl_view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_compare">
          <property name="text">
           <string>Compare</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_open"/>
    <addaction name="act_save"/>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="mnu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="act_pref"/>
   </widget>
   <widget class="QMenu" name="mnu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="act_support"/>
    <addaction name="act_about"/>
   </widget>
   <addaction name="mnu_file"/>
   <addaction name="mnu_edit"/>
   <addaction name="mnu_help"/>
  </widget>
  <action name="act_exit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="act_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="act_pref">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="act_support">
   <property name="text">
    <string>Support</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="act_open">
   <property name="text">
    <string>Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>QClickDropLabel</class>
   <extends>QLabel</extends>
   <header>qt/QClickDropLabel</header>
  </customwidget>
  <customwidget>
   <class>QImageLabel</class>
   <extends>QLabel</extends>
   <header>qt/QImageLabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>act_exit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
